artcommon/artcommonlib/brew.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class TaskStates(Enum):
```

Comment:
I think these can move to constants or build_util lib

Reply:
There's a lot of repeated code in brew.py in both elliott and doozer. The ideal would be to move all of that in here and keep everything together. But not tackling that big change in this PR.